lumigator/python/mzai/jobs/inference/inference.py

```diff
@@ -86,24 +86,24 @@ def run_inference(config: Box) -> Path:
 
     # Enable / disable tqdm
     input_samples = dataset["examples"]
-    dataset_iterable = tqdm(input_samples) if config.evaluation.enable_tqdm else input_samples
+    dataset_iterable = tqdm(input_samples) if config.job.enable_tqdm else input_samples
 
     # Choose which model client to use
-    if config.model.inference is not None:
+    if config.server is not None:
```

Comment:
could we call this something else, perhaps `model_client` then?

Reply:
In this case it is actually a server, so we could call it `inference_server`. Wdyt?